packages/prisma/src/prisma/schema.prisma

```diff
@@ -178,13 +178,15 @@ model Question {
 
   name               String
   content            String
-  options            Json
   explanation        String?
   pointsMultiplier   Int                 @default(1)
   displayMode        QuestionDisplayMode @default(LIST)
   hasSampleSolution  Boolean             @default(false)
   hasAnswerFeedbacks Boolean             @default(false)
 
+  /// [PrismaQuestionOptions]
```

Comment:
this syntax will cause the json addon for prisma to replace all generic Json types with the type stated here. the addon works purely on the typescript level and output and does not change any logic.